### ISSUE TITLE
feat: Migrate Starknet : get_transaction_by_hash from "beerus_rest_api" to "beerus_rpc"

### ISSUE THREAD

**Issue by danilowhk** [keep-starknet-strange/beerus#234]
# Feature Request

**Describe the Feature Request**

We are deprecating "beerus_rest_api" and now migrating all the api endpoints to "beerus_rpc".

Migrate get_transaction_by_hash and add tests on beerus_rpc.

**Describe Preferred Solution**

**Describe Alternatives**

**Related Code**

**Additional Context**

**If the feature request is approved, would you be willing to submit a PR?**
_(Help can be provided if you need assistance submitting a PR)_

- [ ] Yes
- [ ] No

**Comment by dlight** [comment]
May I take this one?

**Comment by dlight** [comment]
I think @phklive is doing this already ([here](https://github.com/phklive/beerus/tree/feat/get_transaction_by_hash))

**Comment by phklive** [comment]
No, it is not the same, what i implimented which is in pull request: #201 is the verify function in beerus.rs you need to impliment into beerus-rpc.

**Comment by dlight** [comment]
Oh, got it!

**Comment by phklive** [comment]
Hey @dlight how is it going with this PR do you need any help?

**Comment by phklive** [comment]
Taking on this issue
